app/config/settings.py

```diff
@@ -45,6 +45,7 @@
 
 COMMON_MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```

Comment:
```suggestion
```

This has actually been moved to line 60 when I noticed a bug with the dev env. See also lines 65-68